mlflow/models/evaluation/default_evaluator.py

```diff
@@ -175,7 +171,7 @@ def _get_classifier_global_metrics(is_binomial, y, y_pred, y_probs, labels):
     get classifier metrics which computing over all classes examples.
     """
     metrics = {}
-    metrics["accuracy"] = sk_metrics.accuracy_score(y, y_pred)
+    metrics["accuracy_score"] = sk_metrics.accuracy_score(y, y_pred)
```

Comment:
@yxiong @jimmyxu-db Renaming for consistency with `mlflow.sklearn.eval_and_log_metrics`; let me know if this is unnecessary

Reply:
I think it's nice to be consistent :)